roles/setup_repo/tasks/PG_Debian_setuprepos.yml

```diff
@@ -29,6 +29,14 @@
     state: present
   become: true
 
+# Package is used for ident authentication
+- name: Install oidentd package when install_pgd
+  ansible.builtin.package:
+    name: oidentd
+    state: present
+  become: true
+  when: install_pgd
```

Comment:
The install shouldn't happen in the `setup_repo` role

Reply:
And the conditional should be the same as for RedHat
```
  when: 
    - install_pgd  
    - enable_epel_repo
```

Reply:
Where should this piece of code be installed then?

Reply:
I feel like this should at least be near where ever we define the postgres packages to install.